### PR TITLE
test: replace cypress tests to open in no iframe FE-4002

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Open tests in no Iframe if possible
Clean the code removing old steps which are no used anymore.
### Current behaviour

We still have cypress tests opening in iframe.
We have old steps which are no used anymore.
### Checklist

- [x] Commits follow our style guide
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

File: cypress/features/regression/alertActions.feature [hunk 1]
```diff
@@ -1,25 +1,9 @@
...
-  @positive
-  Scenario: Clicking close icon, closes Alert dialog
-    # commented because of BDD default scenario Given - When - Then
-    # When I open "Alert Test" component page "default"
-    Then closeIcon is visible in iframe
-      And I click closeIcon in IFrame
-      And Alert is not visible
-
-  @positive
-  Scenario: Disable escape key
```
Comment (DlgSHi) [inline cypress/features/regression/alertActions.feature:16]: why we deleted this TC ?

File: cypress/features/regression/experimental/checkbox.feature [hunk 2]
```diff
@@ -31,6 +31,6 @@ Feature: Experimental Checkbox component
 
   @positive
   Scenario: Change Checkbox tick color
-    Given I open "Experimental Checkbox Test" component page "default"
+    When I open Default "Experimental Checkbox Test" component in noIFrame with "checkbox" json from "experimental" using "default" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:34]:
```suggestion
    Given I open Default "Experimental Checkbox Test" component in noIFrame with "checkbox" json from "experimental" using "default" object name
```

File: cypress/features/regression/experimental/dateInput.feature [hunk 3]
```diff
@@ -52,34 +52,34 @@ Feature: Experimental Date Input component
 
   @positive
   Scenario: Open dayPickerDay via click on input
-    Given I open "Experimental Date Input Test" component page "default"
-    When I click dateInput
+    Given I open default "Experimental Date Input Test" component in noIFrame with "dateInput" json from "experimental" using "default" object name
+    When I click dateInput in noIframe
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateInput.feature:56]: we have only in this line `in noIframe`, so we need to use everywhere with `in noiFrame` or remove it from this line and replace it with proper step

File: cypress/fixtures/commonComponents/dialog.json [hunk 4]
```diff
@@ -53,7 +53,19 @@
   "showCloseIconFalse": {
     "showCloseIcon": false
   },
+  "showCloseIconTrue": {
```
Comment (DlgSHi) [inline cypress/fixtures/commonComponents/dialog.json:56]:
```suggestion
  "showCloseIcon": {
```

**Review by DlgSHi — COMMENTED** [review]
do we need those 
```
const TEXT_ALIGN = "justify-content";
const TEXT_ALIGN_START = "flex-start";
const TEXT_ALIGN_END = "flex-end";
```
in `common-steps.js` file

File: cypress/features/regression/alertActions.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/alertActions.feature:16]: it's the same TC as in `alert.feature` file

File: cypress/support/step-definitions/common-steps.js [hunk 5]
```diff
@@ -11,45 +11,34 @@ import {
...
-  dlsRoot,
 } from "../../locators";
 import { dialogTitle } from "../../locators/dialog";
 import DEBUG_FLAG from "..";
 import { pagerSummary } from "../../locators/pager";
 
 const TEXT_ALIGN = "justify-content";
-const TEXT_ALIGN_START = "flex-start";
-const TEXT_ALIGN_END = "flex-end";
+// const TEXT_ALIGN_START = "flex-start";
```
Comment (DlgSHi) [inline cypress/support/step-definitions/common-steps.js:40]: so, do we need those or not :D

Comment (KatarzynaQA) [reply cypress/support/step-definitions/common-steps.js]: no, removed

**Review by DlgSHi — APPROVED** [review]

**Pull request merged**